src/externalServices/storeTrigger/requestCall.ts

```diff
@@ -30,4 +31,23 @@ export class StoreTriggerCall {
     });
     return response.data;
   }
+
+  public async deletePayload(request: StoreTriggerDeletePayload): Promise<StoreTriggerResponse> {
+    this.logger.debug({
+      msg: 'sending the delete request to store-trigger',
+      modelId: request.modelId,
+      pathToTileset: request.pathToTileset,
+      modelName: request.modelName,
+    });
+    const response = await axios.post<StoreTriggerResponse>(`${this.storeTrigger.url}/jobs/delete`, request);
+    this.logger.info({
+      msg: 'sent to store-trigger successfully',
+      jobId: response.data.jobID,
+      modelId: request.modelId,
+      pathToTileset: request.pathToTileset,
+      modelName: request.modelName,
+      request,
```

Comment:
You already print the content of "request", there is no need to print it

Reply:
ok